sample/App.fs

```diff
@@ -49,10 +61,30 @@ let view model dispatch =
 // |> Program.run
 
 [<LitElement("sample-app")>]
-let App() =
-    Hook.useHmr(hmr)
-    let _ = LitElement.init(fun config ->
-        config.useShadowDom <- false
-    )
-    let model, dispatch = Hook.useElmish(init, update)
-    view model dispatch
\ No newline at end of file
+let App () =
+    Hook.useHmr (hmr)
+    let _ = LitElement.init (fun config -> config.useShadowDom <- false)
+    let model, dispatch = Hook.useElmish (init, update)
+    view model dispatch
+
+[<Emit("String")>]
+let StringCtor: obj = jsNative
+
+registerElement "user-profile" JsInterop.jsConstructor<ClassComponents.UserProfile> {
+    property "name" {
+        use_attribute
+        use_type StringCtor
+    }
+
+    property "age" {
+        use_attribute
+        use_type JS.Constructors.Number
+    }
+
+    css $"p {{ color: red; }}"
+}
+
+registerElement "element-with-controller" JsInterop.jsConstructor<ClassComponents.ElementWithController> {
+    css $"p {{ color: red; }}"
+    css $"li {{ color: rebeccapurple; }}"
```

Comment:
A bit concerned that the class and the registration (with the props, styles...) are so far away so the compiler cannot tell you when you misspell a prop name. But let's see how it goes (it's that compiler doesn't the prop names of html templates either).

Reply:
I don't think this is too far fetched from what current lit users do, but I share this concern arguably I think it would be nice if we stick slightly more close to lit's semantics so people don't have to learn Fable.Lit + Lit